src/components/CustomDialogTitle.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from "@emotion/styled";
+import { Close as CloseIcon } from "@mui/icons-material";
+import Title from "@mui/icons-material/Title";
+import { Button, DialogTitle } from "@mui/material";
+
+interface CustomDialogTitleProps {
+  title: string;
+  subTitle: string;
+  onClose?: () => void;
+}
+
+export const CustomDialogTitle = ({
+  title,
+  subTitle,
+  onClose,
+  ...props
+}: CustomDialogTitleProps) => {
+  return (
+    <DialogTitle {...props}>
+      {onClose ? (
+        <Button>
+          <CloseIcon />
+        </Button>
+      ) : null}
```

Comment:
You dont hv to return null, can do:
{onClose && (
        <Button onClick={onClose} aria-label="Close">
          <CloseIcon />
        </Button>
      )}